adafruit_azureiot/base64.py

```diff
@@ -46,8 +46,8 @@
 def _bytes_from_decode_data(data: str):
     try:
         return data.encode("ascii")
-    except:
-        raise ValueError("string argument should contain only ASCII characters")
+    except Exception as exc:
```

Comment:
`as e` please

Reply:
I originally had e but the lingering the GitHub action wanted more characters :/

Reply:
Ok, pylint does specify that.